package.json

```diff
@@ -20,18 +16,22 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.6.2",
+    "@rails/webpacker": "3.5",
     "babel-loader": "^8.0.6",
+    "prop-types": "~15.6.0",
+    "react": "~16.8.0",
+    "react-dom": "~16.8.0",
     "react-router-dom": "5.0.0",
     "redbox-react": "1.6.0"
   },
   "devDependencies": {
-    "webpack-dev-server": "^2.11.1",
-    "webpack-cli": "^2.0.10",
     "babel-jest": "^24.9.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "~1.14.0",
     "fetch-mock": "~5.13.1",
     "jest": "^24.9.0",
-    "enzyme": "~3.10.0",
-    "enzyme-adapter-react-16": "~1.14.0"
+    "webpack-cli": "^2.0.10",
+    "webpack-dev-server": "^2.11.1"
   },
   "scripts": {
     "start": "./bin/webpack-dev-server",
```

Comment:
remove these changes. While they dont technically change anything, we dont want to change anything having to do with core config